docs/README.md

```diff
@@ -0,0 +1,7 @@
+# API documentation
+
+Generated using [jsdoc](https://github.com/jsdoc3/jsdoc).
+
+**If you are looking for:**
+  * Help to contribute or the general architecture of the project, please have a look at [our wiki](https://github.com/zaproxy/front-end-tracker/wiki).
```

Comment:
The home page contents could be added to a `CONTRIBUTING.md` file in the root dir of the project instead, it has better integration (https://help.github.com/articles/setting-guidelines-for-repository-contributors/) and easier to spot.

Reply:
Opened #7 .